### ISSUE TITLE
Using <ptr> tag generate error "Found no source file"

### ISSUE THREAD

**Issue by paolomarozzi** [evt-project/evt-viewer-angular#255]
Hi, we're trying to move our TEI document that works correctly in EVT 1.3 into Angular 3.0 version (latest beta). 

We setup a dubby domain to host our project but we note that if in the xml file we use the tag `<ptr>` is not rendered by viewer.

Es: 
`<ptr target="#note1" type="noteAnchor"/> `

and below..
```
<div type="notes">
        <note n="1" xml:id="note1"><emph>Segue annotazione di mano posteriore.</emph></note>
```
............

-> we got an "Found no source file" (due to type="noteAnchor" we think)

We know that we can/should use instead:
```
<note place="foot" n="a" xml:id="note1">
              <term xml:lang="latin"></term>
              <emph>Segue</emph> hospitalis <emph>ripetuto. </emph>
</note>
```

Is it a render bug or we're not using the right tag?

Thanks

Paolo

**Comment by RenatoCaenaro** [comment]
@paolomarozzi i updated your comment to display your example xml as 'code'

**Comment by RobertoRDT** [comment]
I suspect that `<ptr/>` is not yet supported in EVT 3, @RenatoCaenaro can you confirm?

**Comment by RenatoCaenaro** [comment]
@RobertoRDT we are going to support `<ptr>` for `<listranspose>` to resolve #273.

You want also support `<ptr>` as a "pointer" to the a footnote? In that case we have to display in the text where `<ptr>` is parsed a baloon/exponent like what we are doing for apps in depa.

**Comment by paolomarozzi** [comment]
Yes.. it will be fantastic to have the text between <note> and </note> shown in a baloon/exponent style.. 

Thanks!!!

**Comment by RobertoRDT** [comment]
Hi @RenatoCaenaro, that would be great, we had that in EVT 1 and it was very convenient for "invariant notes" (= notes whose content applies to different text spans). The syntax proposed by @paolomarozzi looks good (`<ptr target="#note1" type="noteAnchor"/>`) because it makes its purpose explicit, but it should also work without a type when it links to a `<note>` and possibly other similar elements.

We should also think about other uses of `<ptr/>`, in particular when it is just a link to a web site.
